cmd/amp2response.cpp

```diff
@@ -57,7 +57,7 @@ void usage() {
   ARGUMENTS
     + Argument ("amps", "the amplitudes image").type_image_in()
     + Argument ("mask", "the mask containing the voxels from which to estimate the response function").type_image_in()
-    + Argument ("directions", "a 4D image containing the estimated fibre directions").type_image_in()
+    + Argument ("directions_image", "a 4D image containing the estimated fibre directions").type_image_in()
```

Comment:
I changed this to avoid a clash with an option of the same name

Reply:
(Hid / resolved prior post about the same issue)

Issue here is that the *argument* "`directions`" and the *command-line option* "`directions`" encode two very different pieces of information; so disambiguating by calling one an "image" may not be the optimal change. I'm hesitant to change the command-line option, as there are many commands relating to "`amp`" images that have an option "`-directions`" (even if it's not currently centralised with a `const Option` in `core/` / `src/`). Conversely, the functionality of the positional argument here also appears in other commands; so renaming it to something different only here would also be slightly unusual.

A few possibilities come to mind:

1.  Across various commands (thinking `dwi2response` in particular), where an image encoding a unit 3-vector per voxel is used, adopt "`orientations`" rather than "`directions`".

2.  While it's technically applicable to the data type in terms of command naming convention, "`peaks`" would likely not be a good choice here, since can't have multiple vectors per voxel, and the data may have been derived using some approach other than ODF peak-finding.

3.  Rename here (and preferably in other comparable instances also) to "`fibre_directions`".

4.  Create a function that returns an `Option`, creating the current "`-directions`" command-line option with a custom help string per use (I'm pretty sure from experience it's the considerable difference in these help strings that has caused there to *not* be centralisation of this command-line option). Once this works, change the command-line option name to something like "`-amp_directions`".
    (We might need to think carefully about our use of the header field `"Directions"` also...)